dev-docs/modules/mobianRtdProvider.md

```diff
@@ -1,15 +1,189 @@
 ---
 layout: page_v2
 title: Mobian Real-Time Data Provider
-display_name: Mobian Prebid Contextual Evaluation
-description: Mobian provides contextual evaluations of pages given a URL, which publishers can use for targeting as an alternative to keyword-based evaluation.
+display_name: Mobian Prebid Brand Safety Evaluation
+description: The Mobian Real-Time Data (RTD) Module is a plug-and-play Prebid.js adapter that is designed to provide Mobian Contextual results on the publisher’s page.
 page_type: module
 module_type: rtd
 module_code : mobianRtdProvider
 enable_download : true
 vendor_specific: false
 sidebarType : 1
 ---
-# Mobian Contextual Module
-Mobian leverages AI to provide contextual measurement, inclusive of brand safety and suitability, as well as complex signals like emotion, sentiment, tonality and more. 
-This real-time data module pings the Mobian Contextual API at pre-bid time so that advertisers can target and align with their choice of content across Mobian's network of partners.
+
+# Mobian Brand Safety Module
+
+## Overview
+
+Module Name: Mobian Rtd Provider
+
+Module Type: Rtd Provider
+
+Maintainer: <rich.rodriguez@themobian.com>
+
+The Mobian Real-Time Data (RTD) Module is a plug-and-play Prebid.js adapter that is designed to provide Mobian Contextual results on the publisher’s page.
+
+## Downloading and Configuring the Mobian RTD module
+
+Navigate to <https://docs.prebid.org/download.html> and check the box labeled Mobian Prebid Contextual Evaluation. If you have installed Prebid.js on your site previously, please be sure to select any other modules and adaptors to suit your needs. When clicking the "Get Prebid.js" button at the bottom of the page, the site will build a version of Prebid.js with all of your selections.
```

Comment:
```suggestion
Navigate to [the prebid download section](/download) and check the box labeled Mobian Prebid Contextual Evaluation. If you have installed Prebid.js on your site previously, please be sure to select any other modules and adaptors to suit your needs. When clicking the "Get Prebid.js" button at the bottom of the page, the site will build a version of Prebid.js with all of your selections.
```

Using relative links makes automatic dead link checking and working in a test environment a lot more convenient :wink: